src/app/page.tsx

```diff
@@ -1,18 +1,59 @@
-import Image from "next/image";
+"use client";
 import styles from "./page.module.css";
 import { Hero } from "@/components/Hero/Hero";
 import { Tagline } from "@/components/Tagline/Tagline";
 import { ClientQuote } from "@/components/ClientQuote/ClientQuote";
-import { Quote } from "../components/Quote/Quote";
 import { About } from "@/components/About/About";
+import { SwiperPlans } from "../components/SwiperPlans/SwiperPlans";
+import { PlanContainer } from "@/components/PlanContainer/PlanContainer";
+import { Footer } from "@/components/Footer/Footer";
+import { useRef } from "react";
+import { useInView } from "react-intersection-observer";
 
 export default function Home() {
+  const option = { threshold: 0.3 };
+  const aboutRef = useRef();
+  const homeRef = useRef();
+  const plansRef = useRef(null);
+  const testimonialsRef = useRef();
+
+  //! do I need these?
+  const [aboutRefView, aboutVis] = useInView(option);
```

Comment:
good idea but there's a simpler solution. you shouldnt need this or scrollIntoView.

you should assign an id to your sections and your href should link to for example `="#about"`

more information: https://stackoverflow.com/questions/2835140/how-do-i-link-to-part-of-a-page-hash